FreeRTOS/Demo/Common/Minimal/semtest.c

```diff
@@ -231,7 +231,10 @@ short sError = pdFALSE, sCheckVariableToUse;
 			a cycle	(deliberately so to test the guarding) so will be starving
 			out lower priority tasks.  Block for some time to allow give lower
 			priority tasks some processor time. */
-			vTaskDelay( pxParameters->xBlockTime * semtstDELAY_FACTOR );
+			if (pxParameters->xBlockTime != (TickType_t) 0)
+			{
+				vTaskDelay(pxParameters->xBlockTime * semtstDELAY_FACTOR);
+			}

```

Comment:
```suggestion
			if( pxParameters->xBlockTime != ( TickType_t ) 0 )
			{
				vTaskDelay( pxParameters->xBlockTime * semtstDELAY_FACTOR );
			}

```

Reply:
> A WIP PR was just merged. My apologies - would you please move to https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS/Demo/ThirdParty/Partner-Supported.
> 
> Is it more involved than just moving files?

well i'm not sure anything here counts as "partner supported"

Reply:
Well, if you do not plan to address the issues reported in this demo, then you can move this to: https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS/Demo/ThirdParty/Community-Supported

We are still in the process of publishing guidelines about where to put contributions. 

Thanks.

Reply:
I've moved it to Community-Supported for now